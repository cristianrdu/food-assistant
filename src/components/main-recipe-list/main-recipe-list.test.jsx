import { shallow } from 'enzyme';
import React from 'react';
import MainRecipeList from './main-recipe-list';


it('expects to render the MainRecipeList component', () => {
    expect(shallow(<MainRecipeList recipes={[]} match={{params: {routeUrl: test}}} />)).toMatchSnapshot()
})

 