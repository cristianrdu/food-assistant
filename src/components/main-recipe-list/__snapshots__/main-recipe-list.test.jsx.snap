// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the MainRecipeList component 1`] = `ShallowWrapper {}`;
